prediction_market_agent/agents/known_outcome_agent/deploy.py

```diff
@@ -28,14 +27,19 @@ def market_is_saturated(market: AgentMarket) -> bool:
 
 
 class DeployableKnownOutcomeAgent(DeployableAgent):
-    model = "gpt-4-1106-preview"
+    model = "gpt-4-turbo-preview"
```

Comment:
I just noticed this as well, this can point to a different model in future without notice, is that desirable?

Reply:
![Screenshot by Dropbox Capture](https://github.com/gnosis/prediction-market-agent/assets/15619339/993d94b5-1b74-4f91-9834-2f5368842297)

Reply:
Hmm good question. Maybe in general not, but I reckon for this case it's good - my reasoning being that otherwise over time the agent will get less and less 'up to date with current events' which I would assume is bad for making future predictions.